books/RayTracingInOneWeekend.html

```diff
@@ -1749,18 +1876,20 @@
 Generating Pixels with Multiple Samples
 ----------------------------------------
 <div class='together'>
-For a given pixel we have several samples within that pixel and send rays through each of the
-samples. The colors of these rays are then averaged:
+For a single pixel composed of multiple samples, we'll select samples from the area surrounding the
+  pixel and average the resulting light (color) values together.
 
-  ![Figure [pixel-samples]: Pixel samples](../images/fig-1.07-pixel-samples.jpg)
+  ![Figure [pixel-samples]: Pixel samples](../images/fig-1.08-pixel-samples.jpg)
 
 </div>
 
-To handle the multi-sampled color computation, we'll update the `write_color()` function. Rather
-than adding in a fractional contribution each time we accumulate more light to the color, just add
-the full color each iteration, and then perform a single divide at the end (by the number of
-samples) when writing out the color. In addition, we'll add a handy utility function to the
-`interval` class: `clamp(x)`, which clamps the value `x` to the interval:
+First we need to update the `write_color()` function to account for the number of samples we use.
+Rather than adding a fractional contribution each time we accumulate more light to the color, we'll
+  just add the full color from each iteration, and then finish with a single division (by the number
+  of samples) at the end, before writing out the color.
```

Comment:
"We need to find the average across all of the samples that we take. We'll just add the full color from each iteration, and then finish with a single division (by the number of samples) at the end, before writing out the color."

Reply:
Done.